src/utils.py

```
@@ -1,62 +1,90 @@
+    # --- Teszt adatok előkészítése ---
+    X_test = X_test / 255.0
+    X_test = X_test.reshape(-1, IMG_SIZE, IMG_SIZE, 1)
 
-    return (X_train, y_train), (X_val, y_val), (X_test, y_test), num_classes
+    # Módosított visszatérés:
+    return (X_train, y_train_cat), (X_val, y_val_cat, y_val_labels), X_test, num_classes, test_filenames
 
 
-def save_evaluation(history, model, X_test, y_test_labels, model_name="model"):
+import cv2
+import numpy as np
```

Ruff (0.14.7)

Redefinition of unused `np` from line 1

Remove definition: `np`

(F811)